skedulr/app/assets/javascripts/manager_employees.js

```diff
@@ -0,0 +1,42 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
+$(document).ready(function() {
+  $('body').on('click', '.add-employee-to-manager', function () {
```

Comment:
Here you're querying for the whole body, it might be better to just query for the class for the on click event